src/main/java/org/jnode/fs/xfs/Superblock.java

```diff
@@ -92,46 +93,82 @@ public class Superblock extends XfsRecord {
      */
     public static final long XFS_SUPER_MAGIC = 0x58465342;
 
-    /**
-     * The logger.
-     */
-    private static final Logger logger = LoggerFactory.getLogger(Superblock.class);
+    // There is no need to add javadoc on the fields below,
+    // because that will make this file very big and hard to read.
+    // Please check the document for the definition.
```

Comment:
We do however lose the correlation between the variable name in this class and the variable name in the structure. Eg. if we were to look up `realTimeSummaryInode` from Engine, there is no javadoc hint - if we had the Engine project open and we were to try to find the description for `realTimeSummaryInode` we'd have to open this class to find the corresponding struct name - `sb-rsumino`, then search the XFS docs for that description. Very tedious.
It could have been possible to copy/paste the existing javadoc from the previously declared functions.

Reply:
I tried it initially, it just makes the class too big... OK, I'll add the javadoc back :)